### PR TITLE
DDCE-2114: `national-insurance-des-stub` Play 2.8 upgrade

### PR DESCRIPTION
**Pull request by JMaio**
Jenkins build: 
https://build.tax.service.gov.uk/job/DDCE%20Live%20Services/job/SA-PrePop-APIs/job/national-insurance-des-stub/6/

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -13,22 +13,22 @@ def unitFilter(name: String): Boolean = name startsWith "unit"
...
   "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full
 )
 
 def test(scope: String = "test, it") = Seq(
-  "uk.gov.hmrc" %% "reactivemongo-test" % "4.22.0-play-27" % scope,
+  "uk.gov.hmrc" %% "reactivemongo-test" % "5.0.0-play-28" % scope,
   "org.scalatest" %% "scalatest" % "3.0.9" % scope,
-  "org.scalatestplus.play" %% "scalatestplus-play" % "4.0.0" % scope,
-  "org.mockito" % "mockito-core" % "2.10.0" % scope,
+  "org.scalatestplus.play" %% "scalatestplus-play" % "4.0.3" % scope,
```
Comment (kuldeepwadhwa9) [inline build.sbt:27]: Could you upgrade this version to 5.1.0

**Review by kuldeepwadhwa9 — CHANGES_REQUESTED** [review]

File: build.sbt (hunk 1 above)
Comment (JMaio) [reply build.sbt:27]: v5.1.0 requires additional changes to testing, currently erroring with: 
```
[error] national-insurance-des-stub/test/unit/controllers/NationalInsuranceSummaryControllerSpec.scala:24:26: object mockito is not a member of package org.scalatestplus
[error] import org.scalatestplus.mockito.MockitoSugar
```

Comment (JMaio) [reply build.sbt]: updated to `v5.1.0` and issues fixed 👍

**Review by kuldeepwadhwa9 — APPROVED** [review]
Looks good to me :)

**Comment by JMaio** [comment]
re-running Jenkins build, was having a version tagging issue: 
https://build.tax.service.gov.uk/job/DDCE%20Live%20Services/job/SA-PrePop-APIs/job/national-insurance-des-stub/8/

**Pull request merged**